flash-patcher.py

```diff
@@ -140,16 +175,22 @@ def addInjectionTarget(self, injection_info, patch_file, current_line_no):
         self.fileContents[file_name] = file_content
         return file_name
 
-    def addInjectionLine(self, line, current_line_no):
+    def add_injection_line(
+        self: CodeInjector,
+        line: str,
+        current_line_no: int,
+    ) -> None:
+        """Add a line to be injected."""
```

Comment:
nit: the comment should explain that the injection will not actually happen until `inject` is called, as we are only queuing the injection